tasks/debian_packages.yml

```diff
@@ -1,7 +1,24 @@
 ---
 - include_tasks: "{{ ansible_distribution | lower }}_repositories.yml"
 
+- name: Install linux headers (Ubuntu)
+  apt:
+    name:
+      - linux-headers-generic
+      - linux-headers-{{ ansible_kernel }}
+    state: present
+    update_cache: yes
+  when: ansible_distribution == "Ubuntu"
+
+- name: Install kernel headers (Debian)
+  apt:
```

Comment:
This fails on obscure devices with

`fatal: [c2]: FAILED! => {"changed": false, "msg": "No package matching 'linux-headers-4.18.8-odroidc2' is available"}
`